keyboards/mwstudio/mw75r2/keymaps/via/keymap.c

```diff
@@ -41,7 +46,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
     ),
 
 	[1] = LAYOUT(
-      _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, USER00,  USER01,  _______,
+      _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, QK_KB_0,  QK_KB_1,  _______,
```

Comment:
Consistency with other usages
```suggestion
      _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, USER00,  USER01,  _______,
```